### DISCUSSION TITLE
Is it possible to add a paper background image as a pdf file?

### DISCUSSION THREAD

**Discussion by Lijesh-k** [clientIO/joint#1995] [Ideas]
Is it possible to add a paper background image as a pdf file?

**Comment by kumilingus** [comment]
I am not sure I understand since there is no PDF export plugin part of JointJS+. But these links might help.

- 3rd-party PDF export example: https://codepen.io/jointjs/pen/NWMjeBm
- how to export PNG including the paper background image: https://github.com/clientIO/joint/issues/1502

**Reply by Lijesh-k** [reply]
Thank you @kumilingus
